app/org/apache/spark/deploy/history/SparkFSFetcher.scala

```diff
@@ -70,12 +69,7 @@ class SparkFSFetcher(fetcherConfData: FetcherConfigurationData) extends Elephant
    * properties in the configuration files. Triggering it too early will sometimes make the configuration object empty.
    */
   private lazy val _logDir: String = {
-    val conf = new Configuration()
```

Comment:
Sorry for the late reply. I'd like to do so before I witness this situation:
When deploy on a product cluster, the analysis speed is slower than the jobs finish speed! Our cluster runs around 20 thousands jobs every day. But dr. elephant can only analysis around 14 thousands of them. 
I have increased the consumer thread number to 30 or larger, it didn't help. The cpu usage, memory usage and network usage were still in low rate. I found that the bottleneck is the job history server. If dr. elephant can fetch data directly from HDFS, the analysis rate may increase. 

As for the SparkFSFetcher, if use HDFS Rest API, the namenode will be the bottleneck as well. 

I prefer that we should develop another tow fetchers: One can fetch mapreduce data from HDFS directly without dependence on job history server. The other can fetch spark event logs from HDFS rest api. Then we have four fetchers in two kinds: One kind of them fetch data from HDFS, which has high analysis rate, and is suitable for product cluster. The other kind of them fetch data from rest api of job history server and HDFS namenode, which is suitable for tiny cluster or easy deployment for our new users. 

@akshayrai What do your think about this? Hope for further discussion with you :)